### ISSUE TITLE
Use allowed values for "bibl@type"

### ISSUE THREAD

**Issue by wrznr** [slub/mets-mods2tei#36]
http://dfg-viewer.de/strukturdatenset/ has to be mapped onto http://deutschestextarchiv.de/doku/basisformat/mdSdBibl.html

**Comment by wrznr** [comment]
@cthomasdta FYI

**Comment by bertsky** [comment]
An exhaustive list of tags can be found in [Kitodo.Production](https://github.com/kitodo/kitodo-production/blob/master/Kitodo/rulesets/dfg-viewer.xml) BTW.

**Comment by cthomasdta** [comment]
\+ for DTABf-M we would like to have (additional to `manuscript`/`MAN`) things like 
* draft / Entwurf(shandschrift)
* concept / Konzept 
* fair copy / Reinschrift
* diary / Tagebuch (also a category of printed publications, of course)
* etc.

I think this would be the right place in the header for that kind of categorisation, right?

**Comment by bertsky** [comment]
Thanks @cthomasdta for bringing up this additional case! I did not notice it before but it [is in the schema already](https://github.com/deutschestextarchiv/dtabf/blob/ee9a0d580804c8ec36c127d7fecf986440c7eea1/schema/basisformat_all.odd#L371) – perhaps the documentation website should be updated?

We already have `manuscript` in Strukturdatenset for METS divs, but I wonder where (at what place in the representation) your subcategories would go. Do we have an analogue to that in the existing types?

**Comment by cthomasdta** [comment]
> Thanks @cthomasdta for bringing up this additional case! I did not notice it before but it [is in the schema already](https://github.com/deutschestextarchiv/dtabf/blob/ee9a0d580804c8ec36c127d7fecf986440c7eea1/schema/basisformat_all.odd#L371) – perhaps the documentation website should be updated?

Yes, you are right, `<bibl type="MAN">` is missing in the DTABf docu [here](https://www.deutschestextarchiv.de/doku/basisformat/mdSdBibl.html), but it is given in the overview of header elements [here](https://www.deutschestextarchiv.de/doku/basisformat/uebersichtHeader.html).

> We already have `manuscript` in Strukturdatenset for METS divs, but I wonder where (at what place in the representation) your subcategories would go. Do we have an analogue to that in the existing types?

I will think about this (esp. if these are subcategories indeed) and try to find examples.

**Comment by bertsky** [comment]
> * manuscripts: `//fileDesc/sourceDesc/msDesc` -- I would place `"draft"`, `"concept"`, `"fair copy"`, `"diary"` etc. here.

You mean as in
```XML
<msDesc>
  <physDesc>
    <typeDesc>
      <p>diary</p>
    </typeDesc>
  </physDesc>
</msDesc>
```
?

Are you sure that `bibl` is exclusively used for print? IIUC `msDesc` (in TEI literally _(manuscript description) contains a description of a single identifiable manuscript or other text-bearing object such as early printed books._) only describes the original written form of the work. But if the work has not been printed or there is no earlier written exemplar for it, then this should be kept empty.

**Comment by cthomasdta** [comment]
No, I did mean `//fileDesc/sourceDesc/msDesc`, cf. these two examples from edition humboldt digital, where alle 500+ other docs are encoded accordingly -- and hopefully not wrong:
* https://edition-humboldt.de/H0000608.xml
* https://edition-humboldt.de/H0002731.xml

Here, we use `@rend` instead of `@type`, for reasons I cannot remember at the moment, but this is not our issue here.

= `//fileDesc/sourceDesc/msDesc[@rend='manuscript']`, other values are:
* `"notExtant"` | Erschlossen 
* `"concept"` | Entwurf
* `"manuscript"` | Manuskript
* `"copy"` | Abschrift
* `"fragment"` | Fragment
* `"print"` | Druck (e.g. sth. originally printed, but with handwritten notes by our author, therefore treated *as* a manuscript.)

**Comment by bertsky** [comment]
> Here, we use `@rend` instead of `@type`, for reasons I cannot remember at the moment, but this is not our issue here.
> 
> = `//fileDesc/sourceDesc/msDesc[@rend='manuscript']`, other values are:

understood. So in your examples, `bibl type="MAN"` is still missing, and in the DTABf documentation (website), `msDesc/@rend` is.

Also, perhaps `msDesc/@rend` should be restricted by the schema to a set of allowable values (from the Strukturdatenset).

I can find `manuscript` and `fragment` in Strukturdatenset, but not the other values. So should the METS Anwendungsprofil be extended for manuscripts accordingly? (You don't seem to offer METS on edition-humboldt, so perhaps we should consider this a case of #61. You don't happen to know how DTABf relates to the TEI Anwendungsprofil?)

**Comment by cthomasdta** [comment]
> […]
> (You don't seem to offer METS on edition-humboldt, so perhaps we should consider this a case of #61. 

As far as I understand issue 61, yes. That:

> You don't happen to know how DTABf relates to the TEI Anwendungsprofil?)

I do not know, sorry. Maybe @tboenig does?
